ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_ipahealthcheck.py::TestIpaHealthCheck::test_sosreport_includes_healthcheck
```

Comment:
Please run the temp commit with `test_integration/test_ipahealthcheck.py::TestIpaHealthCheck`. Otherwise potential interactions are not detected, and the new test `test_sosreport_without_healthcheck_installed` is not executed.